WordPress/Classes/Models/AbstractPost.swift

```diff
@@ -61,6 +61,26 @@ extension AbstractPost {
         }
     }
 
+    // MARK: - Updating the Remote Status
+
+    /// Updates the post after an upload failure.
+    ///
+    /// - Important: This logic could have been placed in the setter for `remoteStatus`, but it's my belief
+    ///     that our code will be much more resilient if we decouple the act of setting the `remoteStatus` value
+    ///     and the logic behind processing an upload failure.  In fact I think the `remoteStatus` setter should
+    ///     eventually be made private.
+    ///
+    @objc
+    func processUploadFailure() {
```

Comment:
Maybe we can rename this method to `failedToUpload()`?

Reply:
Done.